autoload/lsp.vim

```diff
@@ -502,6 +502,42 @@ def lsp#showCurrentDiag()
   endif
 enddef
 
+# get the count of error in the current buffer
+def lsp#errorCount():dict<number>
+  var res = {'E': 0, 'W': 0, 'I': 0, 'H': 0}
```

Comment:
To make this readable, can you expand this to 'Error', 'Warn', 'Info' and 'Hint'?

Reply:
Done